src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -291,6 +291,41 @@
       usage = "RM3 parameter: turn off English term filter")
   public boolean rm3_noTermFilter = false;
 
+    // --------------------------
+  // query expansion model: rocchio
+  // --------------------------
+
+  // Anserini uses the same default options as in Indri.
```

Comment:
Change this to say that you got these default settings from vector-prf - give cite, etc.

Reply:
Updated the comments. Btw, I found the score is usually better with shorter fbTerms, e.g 20,50. Should we still use the default setting(fbTerms=256) from vector-prf?